infrastructure/templates/template.json

```diff
@@ -3853,6 +3866,61 @@
         "targetResourceLocation": ""
       }
     },
+    {
+      "name": "[parameters('imageTemplateName')]",
+      "type": "Microsoft.VirtualMachineImages/imageTemplates",
+      "apiVersion": "2020-02-14",
+      "location": "[parameters('svcLocation')]",
+      "dependsOn": [],
+      "properties": {
+        "source": {
+          "type": "PlatformImage",
+          "publisher": "Canonical",
+          "offer": "UbuntuServer",
+          "sku": "18.04-LTS",
+          "version": "latest"
+        },
+        "customize": [
+          {
+              "type": "shell",
+              "name": "InstallPackages",
+              "inline": [
+                "apt-get update -y",
+                "apt-get upgrade -y",
+                "apt-get wget -y",
+                "wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb",
+                "sudo apt install -y ./google-chrome-stable_current_amd64.deb",
+                "apt-get install software-properties-common -y",
+                "add-apt-repository ppa:deadsnakes/ppa -y",
+                "apt-get install python3.8 -y",
+                "apt-get install python3-pip -y",
+                "echo alias pip='pip3' > ~/.bashrc",
+                "pip install pipenv -y ",
+                "apt-get install python-setuptools -y",
+                "sudo easy_install pip",
+                "apt-get update -y",
+                "apt-get install python3-pip",
+                "/usr/sbin/waagent -force -deprovision+user && export HISTSIZE=0 && sync"
+              ],
+              "inline_shebang": "/bin/sh -x"
+              
+          }
+        ],
+        "distribute": [
+          {
+            "type": "ManagedImage",
+            "imageId": "[variables('resourceid')]",
+            "location": "[parameters('svcLocation')]",
+            "runOutputName": "[parameters('managedimagename')]",
+            "aritfactTags": {
+              "source": "azVmImageBuilder",
+              "releaseStatus": "production",
+              "type": "mdionly"
+            }
+          }
+        ]
+      }
+    },
```

Comment:
All of these changes should go into their own template file. We want to create a new DevOps pipeline to try and build this image separately from the rest of the infrastructure, so we can't store all of this in the same template file as everything else.

Would suggest putting it in new directory i.e. `infrastructure/templates/test-runner/image-template.json`. If this image builds successfully, we might want to make deployment of the VMs independent of the building of the image by having them in separate template files.